src/wp-includes/block-editor.php

```diff
@@ -269,3 +269,75 @@ function get_block_editor_settings( $editor_name, $custom_settings = array() ) {
 
 	return $editor_settings;
 }
+
+/**
+ * Preloads common data used with the block editor by specifying an array of
+ * REST API paths that will be preloaded for a given block editor context.
+ *
+ * @since 5.8.0
+ *
+ * @global WP_Post $post Global post object.
+ *
+ * @param WP_Block_Editor_Context $block_editor_context The current block editor context.
+ * @param array                   $preload_paths        Optional list of paths to preload.
+ *
+ * @return void
+ */
+function block_editor_preload_api_requests( $block_editor_context, array $preload_paths = array() ) {
```

Comment:
Just a nitpick but... Thinking the params order here doesn't make sense. Which is the "important" param, the paths or the context? Does it make sense (for a plugin) to call this function without any paths? Even if it does, should an empty array be required (seems logical)?

Also in the filter the params are other-way round. Not ideal imho :)

Reply:
Updated in 04c5464.